code/__DEFINES/reactions.dm

```diff
@@ -259,3 +259,9 @@
 #define PN_BZASE_RAD_RANGE_DIVISOR 1.5
 /// A scaling factor for the threshold of the radiation pulses generated when proto-nitrate breaks down BZ.
 #define PN_BZASE_RAD_THRESHOLD_BASE 15
+/// A scaling factor for the nuclear particle production generated when proto-nitrate breaks down BZ.
+#define PN_BZASE_NUCLEAR_PARTICLE_DIVISOR 5
+/// The maximum amount of nuclear particles that can be produced from proto-nitrate breaking down BZ.
+#define PN_BZASE_NUCLEAR_PARTICLE_MAXIMUM 12
```

Comment:
This seems really high, especially since this mix is likely to be on more then one tile at once. Worries me

Reply:
I have no problem with lowering it. Do you have any suggestions for a more appropriate value that's less worrying? I might go with 6 if that's okay.

Reply:
I have reduced it to 6 per gasmixture per atmos tick. I can still lower it if it's still too high.